ui/src/main/webapp/src/app/reports/ignoredFiles/ignoredFiles-report.component.ts

```diff
@@ -0,0 +1,76 @@
+import {Component, OnInit} from "@angular/core";
+import {ActivatedRoute, Params, Router}   from '@angular/router';
+
+import {IgnoredFilesReportService} from "./ignoredFiles-report.service";
+import {NotificationService} from "../../core/notification/notification.service";
+import {utils} from '../../shared/utils';
+import {forkJoin} from "rxjs/observable/forkJoin";
+import {ProjectModel} from "../../generated/tsModels/ProjectModel";
+import {FileModel} from "../../generated/tsModels/FileModel";
+import {TechnologyKeyValuePairModel} from "../../generated/tsModels/TechnologyKeyValuePairModel";
+import {FilterApplication, RegisteredApplication} from "../../generated/windup-services";
+import {IgnoredFileModel} from "../../generated/tsModels/IgnoredFileModel";
+import {WindupVertexFrame} from "../../generated/tsModels/WindupVertexFrame";
+import {BaseModel} from "../../services/graph/base.model";
+
+@Component({
+    selector: 'wu-ignoredFiles-report',
+    templateUrl: 'ignoredFiles-report.component.html',
+    styleUrls: ['../../../../css/report-tables.scss'],
+})
+export class IgnoredFilesReportComponent implements OnInit {
+
+    private execID: number;
+    private ignoredFiles: IgnoredFileModel[] = [];
+
+    private searchText: string;
+
+    private loading = {
+        ignoredFiles: true
+    };
+
+    emptyFilterCallbacks = {
+        ignoredFiles: (ignoredFile) => true,
+    }
+
+    filterCallbacks = Object.assign({}, this.emptyFilterCallbacks);
+
+
+
+    constructor(private route: ActivatedRoute,
+                private ignoredFilesService: IgnoredFilesReportService,
+                private _notificationService: NotificationService,
+                private _router: Router) {
+    }
+
+    ngOnInit(): void {
+        this.route.parent.params.forEach((params: Params) => {
+            this.execID = +params['executionId'];
+            this.fetchIgnoredFiles();
+        });
+    }
+
+    fetchIgnoredFiles(): void {
+        this.ignoredFilesService.getIgnoredFilesInfo(this.execID).subscribe(
+            ignoredFiles => {
+                this.ignoredFiles = ignoredFiles.map(f => <IgnoredFileModel> f);
```

Comment:
That `.map` call is not required. It doesn't do anything. If TypeScript is smart enough, it won't be present in result JS code. If it is not, it will be there, but it won't have any effect.